app/components/CardBet.tsx

```diff
@@ -17,22 +17,22 @@ import {
 } from '@/entities';
 import { redeemPositions } from '@/hooks/contracts';
 import { waitForTransactionReceipt } from 'wagmi/actions';
-import { useState } from 'react';
+import { PropsWithChildren, useState } from 'react';
 import { ModalId, useModal } from '@/context/ModalContext';
 import { TransactionModal } from './TransactionModal';
 import { MarketThumbnail } from './MarketThumbnail';
 import { UserPositionComplete } from '@/app/my-bets/page';
 import { formatValueWithFixedDecimals } from '@/utils';
 
-interface BetProps {
+interface CardBetProps extends PropsWithChildren {
   userPositionComplete: UserPositionComplete;
 }
 
-export const CardBet = ({ userPositionComplete }: BetProps) => {
-  const [txHash, setTxHash] = useState('');
-  const [isTxLoading, setIsTxLoading] = useState(false);
-  const { openModal } = useModal();
-  const config = useConfig();
+export const CardBet = ({ userPositionComplete, children }: CardBetProps) => {
+  const position = new Position(userPositionComplete.position);
+  const outcomeIndex = position.outcomeIndex - 1;
```

Comment:
We should rely on position for getting `outcomeIndex` so we don't have this logic on components

```suggestion
  const outcomeIndex = position.getOutcomeIndex()
```